README.md

```diff
@@ -85,6 +85,11 @@ use of standard NumPy/dask arrays as its underlying data storage.
 The documentation for Iris is available at <https://scitools.org.uk/iris/docs/latest>,
 including a user guide, example code, and gallery.
 
+## Work in progress
+
+A documentation build for the latest development code is visible at
+<https://scitools-docs.github.io/iris/master/index.html>.
+
```

Comment:
In the past we also had a badge in the [README that pointed to the docs built from master](https://github.com/SciTools/iris/blob/962f0159513e6eb08ed1a76fbc8485a31dd584bb/README.md). Could you readd it?

Reply:
ok!